src/Psalm/DocComment.php

```diff
@@ -134,7 +134,7 @@ public static function parse(string $docblock, ?int $line_number = null, bool $p
         $min_indent = 80;
         $indent = 0;
         foreach (array_filter(explode("\n", $docblock)) as $line) {
-            for ($ii = 0; $ii < strlen($line); ++$ii) {
+            for ($ii = 0, $iiMax = strlen($line); $ii < $iiMax; ++$ii) {
                 if ($line[$ii] !== ' ') {
```

Comment:
Well spotted. Why the `+=` though? I'd have replaced the for by  `$indent = strspn($line, ' ');` no?

Reply:
`$indent` is not reset inside the loops. Maybe it was the intent of the original code, but it's not what it was doing. It kept increasing the indent for every initial space of every line.

Reply:
oh yeah, you're right, that's what it does.

I'll ignore this for this PR because it's focused on linting and no-impact changes but I'll create a new PR when this one will be merged to fix this